README.md

```diff
@@ -20,6 +20,24 @@ collections:
 - name: community.vmware
 ```
 
+### Required Python libraries
+
+VMware community collection depends upon following third party libraries:
+
+* [`Pyvmomi`](https://github.com/vmware/pyvmomi)
+* [`vSphere Automation SDK for Python`](https://github.com/vmware/vsphere-automation-sdk-python/)
+
+### Installing required libraries and SDK
+
+Installing collection does not install any required third party Python libraries or SDKs. You need to install the required Python libraries using following command:
+
+    pip install -r ${ANSIBLE_COLLECTIONS_PATHS}/ansible_collections/community/vmware/requirements.txt
```

Comment:
Actually, we've now got a requirements.txt and test-requirements.txt in the root directory of the collection. The CI uses it, so it's safe to say it's up to date.

Reply:
@goneri I think this won't work because `ANSIBLE_COLLECTIONS_PATHS` can contain several directories:

>\# echo ${ANSIBLE_COLLECTIONS_PATHS}
\~/.ansible/collections:/usr/share/ansible/collections
\# cat \~/.ansible/collections/ansible_collections/community/vmware/requirements.txt 
pyvmomi < 7.0.0
git+https://github.com/vmware/vsphere-automation-sdk-python.git ; python_version >= '2.7'
\# pip install -r ${ANSIBLE_COLLECTIONS_PATHS}/ansible_collections/community/vmware/requirements.txt
Could not open requirements file: [Errno 2] No such file or directory: '~/.ansible/collections:/usr/share/ansible/collections/ansible_collections/community/vmware/requirements.txt'

And, anyway, we don't know if `ANSIBLE_COLLECTIONS_PATHS` is even set... I should guess most people won't and live with the default. So we would move from missing documentation to documentation that doesn't work, which is even worse.

Reply:
@mariolenz is right. and this makes me think Galaxy should be able to return the location of the collection, e.g: `ansible-galaxy collection info $collection` or `ansible-galaxy collection path $collection`.

Reply:
@goneri It looks like your changes are still requested. Can you un-request them? Or would it be the job of @Akasurde to actively ignore them?